### ISSUE TITLE
GenericFormatting.MultipleStatementAlignment false positive when assigning values inside an array

### ISSUE THREAD

**Issue by ping-localhost** [squizlabs/PHP_CodeSniffer#2048]
As of https://github.com/squizlabs/PHP_CodeSniffer/commit/860196ad898b8e5e9a7a79c28be470451af43961 this sniff reports an error for the following snippet:
``` PHP
$file = new File('phpunit', [
    $token_1 = new Token(Token::T_WORD, 'foobar', 1, 0),
    $token_2 = new Token(Token::T_WHITESPACE, ' ', 1, 6),
    $token_3 = new Token(Token::T_WORD, 'barbaz', 1, 7),
    $token_4 = new Token(Token::T_SEMICOLON, ';', 1, 13),
]);
```
PHPCBF incorrectly/unexpectedly fixes the snippet to the following:
``` PHP
$file        = new File('phpunit', [
    $token_1 = new Token(Token::T_WORD, 'foobar', 1, 0),
    $token_2 = new Token(Token::T_WHITESPACE, ' ', 1, 6),
    $token_3 = new Token(Token::T_WORD, 'barbaz', 1, 7),
    $token_4 = new Token(Token::T_SEMICOLON, ';', 1, 13),
]);
```

**Comment by gsherwood** [comment]
Thanks for reporting this. I found the problem and have put in a file. It will be in the 3.3.1 release.
